### ISSUE TITLE
A minor issue: Dark mode preference is not working with sudo

### ISSUE THREAD

**Issue by ineuw** [bleachbit/bleachbit#1168]
**Describe the bug**
Bleachbit is set to dark theme but remains light.
The error happens every time.

**Expected behavior**
To be the same shade as the theme

**Desktop:**
 - OS: Linux Mint Cinnamon 20.1
 - BleachBit version 4.2.0

**Comment by abitrolly** [comment]
I can't repeat it with latest version from source. 

```
git clone https://github.com/bleachbit/bleachbit/
cd bleachbit
./bleachbit.py
```

**Comment by ineuw** [comment]
1. If you cannot repeat it, does that mean it doesn't exist? Please see the attached image.
2. I am not a programmer, and your cryptic reply is meaningless to users. Please rephrase your help reply.
3. I am using the standard cinnamon  theme.

![bleachbit, light mode in a dark theme](https://user-images.githubusercontent.com/10367016/120111642-be515100-c140-11eb-9722-c6617db5b759.jpg)

**Comment by ineuw** [comment]
This happens only in the admin version. I assume it's a permissions issue because the user version works fine.

**Comment by ineuw** [comment]
@abitrolly 
> I can't repeat it with latest version from source.
> 
> ```
> git clone https://github.com/bleachbit/bleachbit/
> cd bleachbit
> ./bleachbit.py
> ```

You were not reading my post. I cloned bleachbit just now, and this is the user version. The missing color problem is with the admin version. I also switched between various dark theme settings and the result is always the same.

**Comment by abitrolly** [comment]
@ineuw I hear you. The problem appears only if the BleachBit is run with `sudo`. Right? It makes sense to rename the ticket to reflect that.

**Comment by rados** [comment]
Just FYI I cannot reproduce it on my Debian 10. So most probably it is related with the user's OS / local settings.

**Comment by ineuw** [comment]
I am also certain that it's a Linux Mint issue. I used a variety of dark themes & settings but it had no effect.

**Comment by az0** [comment]
I tested BleachBit version 4.4.0 on a vanilla install of Linux Mint Cinnamon 20.1, and yes, the BleachBit dark theme option has no effect when running as a regular user or as sudo. 

Then, I found a way to get it working. 
1. Open the Linux Mint "Themes" application
2. Click on Controls
3. Switch to Adwaita

Then, the dark theme preference in BleachBit works, so it is a limitation of the theme.

@ineuw Does this help?
